app/src/main/java/com/sun/makeupwindow/data/repository/ProductRepository.kt

```diff
@@ -42,6 +42,17 @@ class ProductRepository private constructor(
         return local.searchProducts(wordSearch, callback)
     }
 
+    override fun searchProductLoadMore(
+        search: Map<String, Int>,
+        callback: OnDataLoadCallback<List<Product>>
+    ) {
+        return local.searchProductLoadMore(search, callback)
+    }
+
+    override fun getTotalProduct(wordSearch: String, callback: OnDataLoadCallback<Int>) {
```

Comment:
```suggestion
    override fun getTotalProducts(wordSearch: String, callback: OnDataLoadCallback<Int>) {
```
nếu trả về số lượng product là kiểu int thì k nên đặt tên getTotalProduct
nên là: ví dụ numberOfProducts